pages/common/nano.md

```diff
@@ -3,10 +3,14 @@
 > Simple, easy to use editor. An enhanced, free Pico clone.
 > More information: <https://nano-editor.org>.
 
-- Start nano in terminal with {filename}:
+- Start nano in terminal with a specified filename:
 
 `nano {{filename}}`
 
+- Start nano in terminal with filename at specified line and column:
```

Comment:
```suggestion
- Start nano opening a file and positioning the cursor at the specified line and column:
```

Reply:
`- Open a file starting at the specified line and column:`

Perhaps this could be more clear, but I'm unsure. The suggestion you made is quite long.

Reply:
I don't think that's too long. It's medium sized. Just "starting at the specified line and column" is strangely worded, the command doesn't "start" anything.